scripts/yosys/logic_equiv_check.tcl

```diff
@@ -18,6 +18,7 @@ if { [info exists ::env(FP_WELLTAP_CELL)] && $::env(FP_WELLTAP_CELL) ne ""} {
 }
 set decap_cell_wildcard "$::env(DECAP_CELL)*"
 set fill_cell_wildcard "$::env(FILL_CELL)*"
+set buff_ef_cell_wildcare sky130_ef_sc_hd__decap*
```

Comment:
1. Typo, you meant `wildcard`
2. This only works for sky130_fd_sc_hd. It's not PDK-generic.

Reply:
1. yes it's a typo.
2. I'll try to make it generic.

Reply:
set ::env(DECAP_CELL) {sky130_ef_sc_hd__decap_12 sky130_fd_sc_hd__decap_8 sky130_fd_sc_hd__decap_6 sky130_fd_sc_hd__decap_4 sky130_fd_sc_hd__decap_3}
The above env variable is taken from config.tcl generated during the run. As you can see it has all decap cells used in the design. Using the command given below:
**hierarchy -generate $decap_cell_wildcard** , where $decap_cell_wildcard = env(DECAP_CELL)
All decap cells should be removed from the design and this command is removing all decap cells present in **env(DECAP_CELL)** env variable but not removing "sky130_ef_sc_hd__decap_12". I have to declare it explicitly and then again use the above-provided command to eliminate this decap cell from design.
Do you know a way to get rid of this behavior?